server/postgres.js

```diff
@@ -17,13 +17,48 @@ Review.init({
   popularity: Sequelize.INTEGER
 }, { sequelize, modelName: 'review' });
 
+const getReviews = (productid) => {
+  return Review.findAll({
+    where: {
+      productid: productid,
+    },
+    benchmark: true
+  });
+};
+
+const postReview = (review) => {
+  return Review.create(review);
+};
+
+const updateReview = (query, vote) => {
+  return Review.findOne(query)
+    .then((review) => Number(review.popularity))
+    .then((popularity) => {
+      if (vote === 'upvote') {
+        popularity++;
+      } else if (vote === 'downvote') {
+        popularity--;
+      }
+      return popularity;
+    })
+    .then((newPopularity) => Review.update(query, {$set: {popularity: newPopularity}}, {new: true}));
+};
+
+const deleteReview = (query) => {
+  return Reviews.remove(query);
+};
+
 sequelize
   .authenticate()
   .then(() => {
-    console.log('Connected to db!')
+    console.log('Connected to PostgreSQL!')
   });
 
 module.exports = {
   sequelize,
-  Review
+  Review,
+  getReviews,
```

Comment:
You can probably rename these functions to `get`, `post`, `update` and `delete`.